connector/json-test-connector/Connector.toml

```diff
@@ -15,8 +15,3 @@ source = true
 image = "infinyon/fluvio-connect-json-test-source:0.1.0"
 
 
-
```

Comment:
Why is this removed? This is needed for schema

Reply:
In this approach, `template` is not a parameter but a mandatory config entry, the same as a topic name, for example.

Reply:
Added back specification of custom config, but now as OpenApi spec

Reply:
Nice!